extension/address/findAddress.js

```diff
@@ -51,7 +51,7 @@ function prepareAddress (address) {
  */
 function matchAddress (addressA, addressB) {
   try {
-    assert.deepEqual(prepareAddress(addressA), prepareAddress(addressB))
+    assert.strict.deepEqual(prepareAddress(addressA), prepareAddress(addressB))
```

Comment:
In a strict mode zipCode `35510` and `"35510"` are not the same. Do we expect it?

Reply:
Non strict is deprecated in the version we use